src/gaurav/Shop.java

```diff
@@ -0,0 +1,70 @@
+package gaurav;
+
+public class Shop {
+	String maggi;
+	int maggiQty;
+	String dosa;
+	int dosaQty;
+	String pouches;
+	int pouchesQty;
+	String panipuri;
+	int panipuriQty;
+	String masala;
+	int masalaQty;
+
+	void initialQty(int maggi, int dosa, int pouches, int panipuri, int masala) {
+		maggiQty = maggi;
+		dosaQty = dosa;
+		pouchesQty = pouches;
+		panipuriQty = panipuri;
+		masalaQty = masala;
+	}
+
+	void requestQty(int maggi, int dosa, int pouches, int panipuri, int masala) {
+		maggiQty = maggiQty - maggi;
```

Comment:
what if request quantity is more than the available?
please handle this scenario.
Refer this code
**https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/167/files**